soc/arm64/nxp_imx/mimx8m/mmu_regions.c

```diff
@@ -25,25 +26,17 @@ static const struct arm_mmu_region mmu_regions[] = {
 			      DT_REG_SIZE(DT_NODELABEL(ccm)),
 			      MT_DEVICE_nGnRnE | MT_P_RW_U_NA | MT_NS),
 
-	MMU_REGION_FLAT_ENTRY("UART2",
-			      DT_REG_ADDR(DT_NODELABEL(uart2)),
-			      DT_REG_SIZE(DT_NODELABEL(uart2)),
-			      MT_DEVICE_nGnRnE | MT_P_RW_U_NA | MT_NS),
-
 	MMU_REGION_FLAT_ENTRY("ANA_PLL",
 			      DT_REG_ADDR(DT_NODELABEL(ana_pll)),
 			      DT_REG_SIZE(DT_NODELABEL(ana_pll)),
 			      MT_DEVICE_nGnRnE | MT_P_RW_U_NA | MT_NS),
 
-	MMU_REGION_FLAT_ENTRY("UART4",
-			      DT_REG_ADDR(DT_NODELABEL(uart4)),
-			      DT_REG_SIZE(DT_NODELABEL(uart4)),
-			      MT_DEVICE_nGnRnE | MT_P_RW_U_NA | MT_NS),
-
 	MMU_REGION_FLAT_ENTRY("IOMUXC",
 			      DT_REG_ADDR(DT_NODELABEL(iomuxc)),
 			      DT_REG_SIZE(DT_NODELABEL(iomuxc)),
 			      MT_DEVICE_nGnRnE | MT_P_RW_U_NA | MT_NS),
+
+	LINKER_DT_REGION_MMU(MMU_REGION_FLAT_ENTRY)
```

Comment:
This should actually be added to all the `mmu_regions.c` files for the ARM64 arch.

Reply:
Done